sys-fs/btrfsmaintenance/btrfsmaintenance-0.3.1.ebuild

```diff
@@ -0,0 +1,45 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+inherit eutils
+
+DESCRIPTION="Scripts for btrfs maintenance tasks like periodic scrub, balance, trim or defrag"
+HOMEPAGE="https://github.com/kdave/btrfsmaintenance"
+SRC_URI="https://github.com/kdave/btrfsmaintenance/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+DEPEND=""
+RDEPEND="${DEPEND}
+	sys-fs/btrfs-progs
+	virtual/cron"
+
+src_install() {
+	dodoc README.md CONTRIBUTING.md btrfsmaintenance.changes
+	insinto /etc/default
+	newins sysconfig.btrfsmaintenance btrfsmaintenance
+	insinto /usr/share/btrfsmaintenance
+	doins btrfsmaintenance-functions
+	exeinto /usr/share/btrfsmaintenance
+	doexe btrfs*.sh
+	insinto /usr/lib/systemd/system
+	doins btrfsmaintenance-refresh.service
+}
+
+pkg_postinst() {
+	elog "Installing default btrfsmaintenance scripts"
+	"${EROOT%/}"/usr/share/btrfsmaintenance/btrfsmaintenance-refresh-cron.sh || die
+	elog "Now edit cron periods and mount points in /etc/default/btrfsmaintenance"
+	elog "then run /usr/share/btrfsmaintenance/btrfsmaintenance-refresh-cron.sh to"
+	elog "update cron symlinks"
+}
+
+pkg_prerm() {
+	# uninstall symlinks
+	elog "Removing symlinks from btrfsmaintenance cron tasks"
+	"${EROOT%/}"/usr/share/btrfsmaintenance/btrfsmaintenance-refresh-cron.sh uninstall || die
```

Comment:
Cron scripts in `/etc/cron.daily` `/etc/cron.weekly` etc. are executed indirectly from `/bin/run-parts` or `/usr/sbin/run-crons` - not by cron directly. So no cron daemon will be confused. 
But - checking and uninstalling the symlinks only in case of final uninstall is better, of course. I will alter the ebuild and test it.